server/client-config/routes.js

```diff
@@ -12,4 +34,23 @@ module.exports = app => {
     res.type('js')
     res.send(response)
   })
+
+  app.get('/ping', nocache, async (req, res) => {
```

Comment:
I'm not sure that the `client-config` package is the right place for this. I appreciate that having the server code split into packages like this makes it tricky to know where to put little things like this. I would probably put it in `xpub-server`.

Reply:
Agreed, given that it's `xpub-server` which is dependant on the database and S3.